kernel/examples/common/Cargo.toml

```diff
@@ -1,7 +1,13 @@
 [package]
 name = "common"
-version = "0.1.0"
-edition = "2021"
+edition.workspace = true
```

Comment:
For other reviewers, [here](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-package-table) are the relevant docs.